src/cljs/org/broadinstitute/firecloud_ui/page/method_repo/method_config_importer.cljs

```diff
@@ -242,17 +246,29 @@
       :else
       [table/Table
        {:columns [{:header "Type" :starting-width 100}
-                  {:header "Item" :starting-width 450
-                   :sort-by (fn [m]
-                              [(m "namespace") (m "name") (int (m "snapshotId"))])
-                   :filter-by (fn [m]
-                                (clojure.string/join "/" (map m ["namespace" "name" "snapshotId"])))
+                  {:header "Namespace" :starting-width 160
+                   :sort-by (fn [m] [(clojure.string/lower-case (m "namespace"))])
```

Comment:
You have an extra set of brackets here that's unnecessary.  You're ending up with a vector of one string, which works because vectors sort by their elements.  But this should be:

    (fn [m] (clojure.string/lower-case (m "namespace")))

Or:

    #(clojure.string/lower-case (% "namespace"))